python/paddle/nn/functional/flash_attention.py

```diff
@@ -407,4 +407,57 @@ def flash_attn_unpadded(
     return out, softmax if return_softmax else None
 
 
-scaled_dot_product_attention = flash_attention
+def scaled_dot_product_attention(
+    query, key, value, attn_mask=None, dropout_p=0.0, is_causal=False
```

Comment:
In order to be consistent with other APIs, there must be a parameter `name=None` at last